iothub/service/tests/Jobs/JobQueryOptionsTests.cs

```diff
@@ -0,0 +1,39 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using FluentAssertions;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace Microsoft.Azure.Devices.Tests.Jobs
+{
+    [TestClass]
+    [TestCategory("Unit")]
+    public class JobQueryOptionsTests
+    {
+        [TestMethod]
+        public void JobQueryOptions_FieldValueInitializations()
+        {
+            // arrange
+            var jobType = new JobType();
+            var jobStatus = new JobStatus();
```

Comment:
I suggest moving this down to the `// rearrange` section so they are only created when they are needed.